src/Libraries/DSCPython/CPythonEvaluator.cs

```diff
@@ -245,9 +245,10 @@ static CPythonEvaluator()
         private static void ProcessAdditionalBindings(PyScope scope, IList bindingNames, IList bindingValues)
         {
             string nodeName;
-            if (!bindingNames[0].Equals("Name"))
+            if (bindingNames.Count == 0 || !bindingNames[0].Equals("Name"))
```

Comment:
Would you define the "Name" here as a const string or something, it seems we do not have reference to node model here so seems no way to reference the property itself.

Reply:
Sure, I can do that.